.fossa.yml

```diff
@@ -8,7 +8,7 @@ project:
 # ignore a noisy warning produced by this file, it's a schema not a package.json
 targets:
   exclude:
-    - type: projectjson
+    - type: projectassetjson
```

Comment:
I think it's supposed to be `npm` -- but these might also be the legacy names since it looks like they rewrote them a bunch recently https://github.com/fossas/fossa-cli/blob/3c6a32e862216828c68f30f98c089b725862c2c8/docs/references/files/fossa-yml.v3.schema.json#L175

Reply:
actually looks like the schema is just wrong

https://github.com/fossas/fossa-cli/blob/3c6a32e862216828c68f30f98c089b725862c2c8/docs/references/files/fossa-yml.v3.schema.json#L175

Reply:
Does the previous code work correctly?

Reply:
yeah -- though I think fossa is globally disabled at the moment

Reply:
Should I update the PR to remove this file?

Reply:
I would ask @getsentry/open-source what their plans are for this

Reply:
FOSSA has been enabled for several months now, so would prefer is the file wasn't deleted. If we delete these few lines, does the `Enforce License Compliance` job run without errors? Seeing as it was added here:
https://github.com/getsentry/sentry/pull/35841